xml/System.Net.Http/HttpVersionPolicy.xml

```diff
@@ -34,7 +34,15 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Only use the requested version. Throws an <see cref="HttpRequestException" /> if a connection with the exact version cannot be established.</summary>
```

Comment:
The summary here suggests that a possible exception is missing in the doc? If you add a `/// <exception><exception/>` comment in the src, the DocsPort tool will add the exception here correctly :)

Reply:
This is an enum value, it's only mentioning that if the value is consumed, the API that uses it will throw.
So this is fine as it is.

Reply:
This is an enum value, so the exception should not be mentioned here. The exception should be mentioned in the API that consumes this enum value:

```suggestion
        <summary>Only use the requested version.</summary>
```